.github/workflows/pr-stable-build.yml

```diff
@@ -0,0 +1,33 @@
+# allowing test inside PRs
+
+name: "PR build release"
+on:
+    push:
+        branches:
+          - '**'        # matches every branch
+          - '!master'   # excludes master
+
+env:
+    CI: false
+jobs:
+    publish-tauri:
+        strategy:
+            fail-fast: false
+            matrix:
+                platform: [macos-latest, ubuntu-20.04, windows-latest]
+        runs-on: ${{ matrix.platform }}
+        steps:
+            - uses: actions/checkout@v3
+            - name: setup node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
```

Comment:
We should probably not use node 16, as this version is quite old now. We ca bump this to node LTS (20).

```suggestion
                  node-version: 20
```